hack/lib/golang.sh

```diff
@@ -932,3 +932,11 @@ kube::golang::build_binaries() {
     fi
   )
 }
+
+# GOMAXPROCS by default does not reflect the number of cpu(s) available
+# when running in a container, please see https://github.com/golang/go/issues/33803
+if command -v python3 &>/dev/null; then
+    GOMAXPROCS=${GOMAXPROCS:-$(python3 "${KUBE_ROOT}/hack/lib/cpu.py")}
```

Comment:
Log what GOPROCS ends up being, even if usnet? It would be useful in determining if this fails for some reason.

Reply:
will do!